### ISSUE TITLE
Failed to open USB device on windows 10 device.

### ISSUE THREAD

**Issue by PREMjITHT** [NXPmicro/mfgtools#376]
Hi @nxpfrankli,

The programming failed due to the Failure in opening the USB device on Windows 10 device.
Opened Windows PowerShell with admin rights,
The device is detected(imx6s), but it showing the below-mentioned error while programming.

FYI: The same UUU tool is working on the Linux machine.

Please find the attached screenshots,

![uuu1](https://github.com/nxp-imx/mfgtools/assets/34344962/6a76131a-4885-4f3f-ba53-2dde1eb75154)
![uuu2](https://github.com/nxp-imx/mfgtools/assets/34344962/db856e96-3077-479e-acce-91bceb297bf5)

**Comment by PREMjITHT** [comment]
I am using U-boot 2019.
I am testing the UUU Tool on Windows 10. As per the documentation only in Windows 7, I need to use winusb to install the driver.
FYI: The same UUU Tool is working on Linux host machines.

**Comment by PREMjITHT** [comment]
I checked the WCID driver using Zadig.
Below are the results,

![Zadig-2](https://github.com/nxp-imx/mfgtools/assets/34344962/a0152c21-a5a7-40f2-8ca4-145d0af37a2b)
After installing the WCID driver, the Zadig application is showing that the driver is not installed.

The above scenario is when I connected the serial USB cable to a powered USB Hub(USB 3.0).

I tried connecting using a normal adapter(Type A - Type C), 
That time I am getting an error that "The USB device is not recognized". 

![usb err](https://github.com/nxp-imx/mfgtools/assets/34344962/15f457c2-2b78-4231-aa3a-ea5c4c792bf3)

![Zadig-unknown](https://github.com/nxp-imx/mfgtools/assets/34344962/6d55cdc9-dd32-42ae-8dfe-f235fe5d477a)

**Comment by PREMjITHT** [comment]
Below is the result of uuu.exe -v 

![uuu err](https://github.com/nxp-imx/mfgtools/assets/34344962/d82af300-2fe6-4f24-b256-c334f9a8d4a3)

**Comment by nxpfrankli** [comment]
Can you show full log included uuu cmd line

**Comment by PREMjITHT** [comment]
Please find the requested image attached below,

![uuu err vv](https://github.com/nxp-imx/mfgtools/assets/34344962/4f99a123-5af3-4c66-a3bb-ddd55cc7d09e)

**Comment by PREMjITHT** [comment]
Hi @nxpfrankli ,
The same UUU Tool is working on another Windows machine. 
It has a USB Type-A port. Previously I was using a laptop with a Type C port.

Now the tool is able to detect the OTG and flashing. 
But while programming the rootfs it got stuck, The progress bas was not moving. But this is not happening on Linux machines.

**Comment by nxpfrankli** [comment]
Did you use bz2 as rootfs?

**Comment by PREMjITHT** [comment]
yes. 
Please check the below error snapshot for more information.

![bulk error](https://github.com/nxp-imx/mfgtools/assets/34344962/478b52b2-2ffa-4c93-8c5a-a41bdd1ee865)

Sometimes it works, but sometimes the flashing gets stuck and later throws the above error.

**Comment by nxpfrankli** [comment]
It should not related your windows environment.  The decompress was done at boards.  Maybe boards was out memory and kill tar by linux. 

can you try drain some file system cache before tar?
 
https://www.tecmint.com/clear-ram-memory-cache-buffer-and-swap-space-on-linux/

**Comment by PREMjITHT** [comment]
Okay, I will try this.
But I wonder why this is not happening in the Linux Host. There it works always without any lag/hangs/USB Bulk error.

In the Windows host, I observed that the rootfs programming speed got reduced and stuck at one point, later I got the error mentioned in the previous comment.

**Comment by nxpfrankli** [comment]
Yes, suppose all transferred's data should be the same between linux and windows.  Maybe wireshark trace some usb packages to help diagnosis.  Try clean cache firstly.

**Comment by PREMjITHT** [comment]
sure, Let me try that and get back to you. Thank you.
